tock/hours/float.py

```diff
@@ -13,43 +17,76 @@
 def float_tasks_for_view(user, rp):
     userdata = UserData.objects.get(user=user)
     float_people_id = get_float_people_id(userdata)
+    error_state = False
+    tasks = []
 
     if not float_people_id:
         return None
 
-    float_tasks = get_float_tasks(
-        start_date=rp.start_date,
-        float_people_id=float_people_id,
-        weeks=1
-    )
-    float_holidays = get_float_holidays(
-        start_date=rp.start_date,
-        weeks=2
-    )
-    float_timeoffs = get_float_timeoffs(
-        end_date=rp.end_date,
-        float_people_id=float_people_id,
-        weeks=26
-    )
-    work_days = get_work_days(
-        holidays=float_holidays,
-        start_date=rp.start_date,
-        end_date=rp.end_date
-    )
-    work_hours = get_work_hours(
-        start_date=rp.start_date,
-        end_date=rp.end_date,
-        timeoffs=float_timeoffs,
-        work_days=work_days
-    )
+    try:
+        float_tasks = get_float_tasks(
+            start_date=rp.start_date,
+            float_people_id=float_people_id,
+            weeks=1
+        )
+    except:
```

Comment:
Did you think about handling the exception a bit more and passing it through? something along the lines of 

```
except Exception as e:
    logger.error(e)
```

I think that's right. If not, it's close.

Reply:
I agree, we should be catching specific exception types only.

Reply:
You're right in that this isn't ideal, but at the moment we don't necessarily know what the error will be, or at least I'm not 100% confident we will (by and large I'm seeing `attempting to call a method on NoneType objects`, but again, no guarantee).  As far as the handling itself goes, that may be better, let me give it a shot!

Reply:
And to add a bit more color here:  the handling of the Float API calls in general could be improved quite a bit.  Again, for the sake of time and stability, this is just to shore things up for the interim.